tests/conftest.py

```diff
@@ -15,6 +15,7 @@
 import pytest
 import requests
 from pytest_bdd import given, then
+from selenium.common.exceptions import JavascriptException
```

Comment:
😿 Why they call it "JavascriptException" and not "JavaScriptException"??? but that's out of scope.